components/editor/modules/meta/ui.js

```diff
@@ -271,6 +254,66 @@ const MetaData = ({
           node={node}
         />
         <br />
+        <MetaForm data={seoData} onInputChange={onInputChange} black />
+        <SlugField
+          black
+          label={t(
+            `metaData/field/${isTemplate ? 'repoSlug' : 'slug'}`,
+            undefined,
+            'slug'
+          )}
+          value={node.data.get('slug')}
+          onChange={onInputChange('slug')}
+          isTemplate={isTemplate}
+          icon={
+            <span style={{ display: 'inline-block', paddingTop: 10 }}>
+              <Checkbox
+                checked={node.data.get('autoSlug')}
+                onChange={onInputChange('autoSlug')}
+                black
+              >
+                <span style={{ verticalAlign: 'top' }}>automatisch</span>
+              </Checkbox>
+            </span>
+          }
+        />
+        <br />
+        {!isTemplate && mdastSchema && mdastSchema.getPath && (
+          <Label>
+            {t('metaData/field/slug/note', {
+              base: FRONTEND_BASE_URL
+                ? FRONTEND_BASE_URL.replace(/https?:\/\/(www\.)?/, '')
+                : '',
+              path: previewPath
+            })}
+            <br />
+            {!!dataAsJs.path && (
+              <>
+                {t('metaData/field/slug/pathNote', {
+                  base: FRONTEND_BASE_URL.replace(/https?:\/\/(www\.)?/, ''),
+                  path: dataAsJs.path
+                })}
+                <br />
+              </>
+            )}
+            <br />
+          </Label>
```

Comment:
This feels like a lot of logic in an already very long file – maybe it deserves its own component?